core-java-collections/src/test/java/com/baeldung/enummap/EnumMapBenchmarkLiveTest.java

```diff
@@ -0,0 +1,117 @@
+package com.baeldung.enummap;
+
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+@BenchmarkMode({Mode.AverageTime})
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@Warmup(iterations = 5)
+@Measurement(iterations = 5)
+public class EnumMapBenchmarkLiveTest {
+    public static volatile int _cnt =0;
+
+    @State(Scope.Thread)
+    public static class BenchmarkState {
+        EnumMap<DummyEnum,String> enumMap = new EnumMap<>(DummyEnum.class);
```

Comment:
whitespacing. The reason I point this out is that we want the examples to look professional and well-thought out. While a simple space doesn't mean that you put less thought into it, it will catch ppl's eye.

Reply:
Noted.